### PR TITLE
Adds the IRA shuttle

### PR DESCRIPTION
**Pull request by bsolomon9**
## About The Pull Request
adds the ira shuttle with bombs and nothing else really
image
image

## Why It's Good For The Game
mm bomb shuttle

## Changelog
:cl:
add: Added shuttle
add: Added config
add: Added the provo datum to assistant
/:cl:

### CODE REVIEW

File: voidcrew/code/modules/jobs/job_types/assistant.dm [hunk 1]
```diff
@@ -73,3 +73,13 @@
 	uniform = /obj/item/clothing/under/rank/prisoner
 	shoes = /obj/item/clothing/shoes/sneakers/orange
 	accessory = /obj/item/clothing/accessory/armband/deputy
+
+/datum/outfit/job/assistant/provo
+	name = "Provisional IRA Scientist"
+
+	head = /obj/item/clothing/head/beret/sec/officer{name="IRA beret"}
```
Comment (SuperNovaa41) [inline voidcrew/code/modules/jobs/job_types/assistant.dm]:
```suggestion
	head = /obj/item/clothing/head/beret/sec/officer{name = "IRA beret"}
```
not on my watch

**Review by SuperNovaa41 — APPROVED** [review]
Code is fine other than this 👍

**Comment by Jackriip** [comment]
LGTM

**Comment by Jackriip** [comment]
testmerging

**Comment by dwasint** [comment]
merged

**Pull request merged**